src/argus/notificationprofile/notification_media.py

```diff
@@ -54,17 +65,40 @@ def send(incident, user):
 
 def send_notifications_to_users(incident: Incident):
     # TODO: only send one notification per medium per user
+    LOG.info("Notification: sending incident \"%s\"", incident)
     for profile in NotificationProfile.objects.select_related("user"):
         if profile.incident_fits(incident):
             send_notification(profile.user, profile, incident)
+    else:
+        LOG.info("Notification: no listeners for \"%s\"", incident)
+        return
+    LOG.info("Notification: incident \"%s\" sent!", incident)
+
+
+def background_send_notifications_to_users(incident: Incident):
+    connections.close_all()
+    LOG.info("Notification: backgrounded: about to send incident \"%s\"", incident)
```

Comment:
```suggestion
    LOG.info(f'Notification: sending incident "{incident}"')
    for profile in NotificationProfile.objects.select_related("user"):
        if profile.incident_fits(incident):
            send_notification(profile.user, profile, incident)
    else:
        LOG.info(f'Notification: no listeners for "{incident}"')
        return
    LOG.info(f'Notification: incident "{incident}" sent!')


def background_send_notifications_to_users(incident: Incident):
    connections.close_all()
    LOG.info(f'Notification: backgrounded: about to send incident "{incident}"')
```
See the comment above on string formatting.

Reply:
See previous.